src/Compilers/Core/Portable/Symbols/ILocalSymbol.cs

```diff
@@ -64,5 +64,21 @@ public interface ILocalSymbol : ISymbol
         /// Returns true if the local variable is declared with fixed-pointer-initializer (in unsafe context).
         /// </summary>
         bool IsFixed { get; }
+
+        /// <summary>
+        /// Returns true if this local variable is declared as iteration variable
+        /// </summary>
+        bool IsForEach { get; }
```

Comment:
>IsForEach [](http://example.com/codeflow?start=13&length=9)

It looks like we already exposed an extension method with the same name for VB. Is addition of this property potentially breaking existing user code? Probably not, because I believe VB allows to use ```()``` after properties, but it would be worth adding a test and verify the code works with and without this property.

Reply:
C# consumers could be affected though.